test/functional/test_framework/address.py

```diff
@@ -20,8 +20,11 @@
     sha256,
     taproot_construct,
 )
-from .segwit_addr import encode_segwit_address
 from .util import assert_equal
+from test_framework.segwit_addr import (
+    encode_segwit_address,
```

Comment:
style nit: these should be lexicographically sorted

Reply:
Okay :+1: